src/matlab2tikz.m

```diff
@@ -3530,18 +3523,17 @@ function warnFacetedInterp(m2t)
     if strcmpi(getEnvironment(), 'Octave')
         sData = sData.^2/2;
     end
-    sData = translateMarkerSize(m2t, matlabMarker, sqrt(sData)/2);
+    sData = translateMarkerSize(m2t, rawMarker.style, sqrt(sData)/2);
 
     drawOptions = opts_new();
     if length(cData) == 3
         [m2t, drawOptions] = getScatterOptsOneColor(m2t, h, drawOptions, ...
-                                                markOptions, tikzMarker, ...
-                                                cData, sData, constMarkerkSize);
+                                                rawMarker, cData, sData, ...
+                                                constMarkerkSize);
```

Comment:
Given
```
constMarkerkSize = length(sData) == 1;
```
I think it is alwase false in this branch.  getScatterOptsColormap handles the vector case